nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -500,7 +510,7 @@ NavfnPlanner::printCostmap(const nav2_msgs::msg::Costmap & costmap)
 }
 
 void
-NavfnPlanner::publishEndpoints(const nav2_tasks::ComputePathToPoseCommand::SharedPtr & endpoints)
+NavfnPlanner::publishEndpoints(const geometry_msgs::msg::Pose & start, const geometry_msgs::msg::Pose & goal)
```

Comment:
This line is too long.

Reply:
@orduno Dang. I'll run the linting tools again and update the code.